README.md

```diff
@@ -27,7 +27,27 @@ function mystery(n) {
     }
 }
 ```
+The base case here is incredibly simple to figure out within the first three lines of code. A simple return for if n <= 1 runs in constant time so we have a time complexity of O(1) regardless. Next, the two recursive calls cut the problem size into 1/3 giving us the following recurrence relation: T(n) = 2*(T(n/3)). The next (and perhaps most difficult) part revolves around the behavior of the inner loops. Each of the three may run from 0 to $n^2$ times. As I learned in 2030, the time complexity of this should be $O(n^5)$ as that's what the product of the loops' operations come out to. ($n^2$ for the first, n for the second, and $n^2$ for the third). Finally we see one more recursive call of mystery cutting the problem space into 1/3 again. Putting it all together we get a final reccurence relation of $T(n) = 3*T(n/3) + O(n^5)$.
 
-Add your answer to this markdown file. [This
-page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
-might help with the notation for mathematical expressions.
+Solving the reccurence relation:
+
+$3T(n/3)+n^5$  
+$9T(n/9)+n^5+3(n/3)^5$  
+$27T(n/27)+n^5+3(n/3)^5+9(n/9)^5$  
+
+Which may be simplified down to:  
+
+$27T(n/27)+n^5+(n^5/3^4)+(n^5/9^4)$  
+
+We see a pattern emerges here with every term containing an n^5 in the numerator and each successive term dividing this n^5 by increasing powers of three all raised to the fourth power. Thus, we can represent this behavior with the following sum: $\sum_{n=1}^{i} (1/3)^{4n}$. We can stick an $n^5$ in front of our sum to account for both the first term and numerator of every term that follows:
+
+$n^5 \sum_{n=1}^{i} (1/3)^{4n}$   
```

Comment:
This would mean that the complexity of the function is exponential in the input size. This is not correct.